crates/oxc_codegen/src/sourcemap_builder.rs

```diff
@@ -62,12 +62,11 @@ impl SourcemapBuilder {
     }
 
     pub fn add_source_mapping_for_name(&mut self, output: &[u8], span: Span, name: &str) {
-        // SAFETY: search original string by span.
-        let original_name =
-            unsafe { self.original_source.get_unchecked(span.start as usize..span.end as usize) };
+        let original_name = self.original_source.get(span.start as usize..span.end as usize);
```

Comment:
Maybe we should add a `debug_assert` to deny all invalid span as many as possible in debug build. For release build, we just ignore it because nothing we could do there.